src/games/strategy/engine/lobby/client/ui/MacLobbyWrapper.java

```diff
@@ -13,10 +14,9 @@
   // i think the java validator triggers this
   public static void registerMacShutdownHandler(final LobbyFrame frame) {
     // new Application();
-    final Application application = Application.getApplication();
-    application.addApplicationListener(new ApplicationAdapter() {
+    Application.getApplication().setQuitHandler(new QuitHandler(){
       @Override
-      public void handleQuit(final ApplicationEvent event) {
+      public void handleQuitRequestWith(QuitEvent arg0, QuitResponse arg1) {
```

Comment:
Please rename parameters `arg0` and `arg1`
I recommend 'quitEvent' and 'quitResponse'